templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
-		<p>A tool for tracking the progress of the kittens (and cat{s,z}) in out CodeCatz meetup group</p>
-	</div>
+        <div> 
```

Comment:
Is there any particular reason for having both `<h1>` and `<h2>` wrapped in an extra `<div>`? There is no class defined that would be used for styling.